Vizzuality/layer-manager
# Monorepo architecture

Monorepo architecture with [Lerna](https://lerna.js.org/).

# Testing
`npm install --global lerna // yarn global add lerna`
`rm -rf node_modules && yarn`
`lerna bootstrap`

Check out the root `package.json` to see available commands.

[More info about Lerna commands](https://github.com/lerna/lerna#readme)

Feel free to test and provide any feedback :)

Try to:
- build packages
- watch packages and test them in the sandbox
- publish packages
- ...
